### PR TITLE
Drop cerberus, it seems it's not used anywhere

### PR DESCRIPTION
**Pull request by oz123**

### CODE REVIEW

**Review by matteius — APPROVED** [review]

**Comment by oz123** [comment]
I added 2 more commits, hence the requerst to review again.

File: tests/unit/test_utils_windows_executable.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 
 @pytest.mark.utils
+@pytest.mark.skip_windows
```
Comment (matteius) [inline tests/unit/test_utils_windows_executable.py:17]: Why are we skipping windows tests on windows, what is the back story here?

Comment (oz123) [reply tests/unit/test_utils_windows_executable.py]: Ah. Good catch, I probably meant this:

```
@pytest.mark.skipif(os.name != "nt", reason="Windows file paths tested")
```

**Review by matteius — APPROVED** [review]
The test failure on one of the runners was network/github related.

**Pull request merged**